libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/ZoomInsightConfiguration.tsx

```diff
@@ -0,0 +1,78 @@
+// (C) 2020-2022 GoodData Corporation
+import { IInsightWidget, insightVisualizationUrl } from "@gooddata/sdk-model";
+import { Bubble, BubbleHoverTrigger, Checkbox, Message } from "@gooddata/sdk-ui-kit";
+import includes from "lodash/includes";
+import React, { useState } from "react";
+import { useIntl } from "react-intl";
+import {
+    changeInsightWidgetVisProperties,
+    selectInsightByRef,
+    useDashboardDispatch,
+    useDashboardSelector,
+} from "../../../../model";
+import { VisType, VisualizationTypes } from "@gooddata/sdk-ui";
+
+const supportedVisualizationTypes = [
```

Comment:
I know that it is implemented in the same way in KD, but this hardcoded set of visualization types is not what we want in code base considering pluggable visualization approach in future. 
It would be probably better to include information if given type supports zooming or not directly into visualizationDescriptor.
Probably extend 
https://github.com/gooddata/gooddata-ui-sdk/blob/56c82ac60c010cdbc6f0d989c6c8933d0b31562a/libs/sdk-ui-ext/src/internal/interfaces/VisualizationDescriptor.ts#L109
and set value for individual viz types in their implementation, eg.
https://github.com/gooddata/gooddata-ui-sdk/blob/56c82ac60c010cdbc6f0d989c6c8933d0b31562a/libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/areaChart/AreaChartDescriptor.ts#L70